kinoml/features/klifs.py

```diff
@@ -0,0 +1,135 @@
+"""
```

Comment:
We need to sync with @dominiquesydow here to see if there are synergies with her `klifs_utils` project here. 

Also, this should live in another subpackage.

Reply:
Great, yes! 

`klifs_utils` has been moved (including API improvements) to our [`opencadd`](https://github.com/volkamerlab/opencadd) module (`opencadd.databases.klifs`) and is currently under review (see [PR](https://github.com/volkamerlab/opencadd/pull/44)).

You can take a look at the API in this tutorial:
https://github.com/volkamerlab/opencadd/blob/add_io_klifs_subpockets/docs/tutorials/databases_klifs.ipynb

The local (if you have a local KLIFS download) and remote (using the [KLIFS Swagger API](https://klifs.vu-compmedchem.nl/swagger/)) queries in `opencadd` always return a  `pandas.DataFrame` - I see from your docstring (@jiayeguo, @jaimergp) that you work with "Python object models". I'd be happy to talk about this.

The function in your `kinoml/features/klifs.py` file probably does the same as the following in `opencadd.databases.klifs` (see cell 65 in the tutorial):

```
from opencadd.databases.klifs import setup_remote
remote = setup_remote()
remote.structures.from_structure_pdbs(["3sxr", "1fpu"])  # One or more PDB IDs
# Returns all KLIFS entries (all chains, alternate models) for the input PDB IDs
```